File: XPile 3-17/Programs/test_task1_renames.js

const MAX = 100;
const LABEL = "test";
function main() {
    console.log(MAX);
    console.log(LABEL);
    return 0;
}

function __ul_sort(arr, n) { const sub = arr.slice(0, n).sort((a, b) => a < b ? -1 : a > b ? 1 : 0); for (let i = 0; i < n; i++) arr[i] = sub[i]; }

main();

File: XPile 3-17/Programs/test_task1_renames.py
from dataclasses import dataclass, field

def __ul_sort(arr, n): arr[:n] = sorted(arr[:n])

class __ul_Exception(Exception):
    def __init__(self, type_name, message):
        self.type_name = type_name
        self.message = message
        super().__init__(message)

MAX = 100
LABEL = 'test'

def main():
    print(MAX)
    print(LABEL)
    return 0


if __name__ == "__main__":
    main()
